src/encoded/schemas/file.json

```diff
@@ -388,6 +390,39 @@
                 "revoked",
                 "released"
                 ]
+        },
+        "dbxrefs": {
+            "@type": "@id",
+            "rdfs:subPropertyOf": "rdfs:seeAlso",
+            "title": "External identifiers",
+            "description": "Unique identifiers from external resources.",
+            "type": "array",
+            "default": [],
+            "items": {
+                "title": "External identifier",
+                "description": "A unique identifier from external resource.",
+                "type":  "string",
+                "pattern": "^\\S+:\\S+"
+            }
+        },
+        "pipeline": {
+            "title": "Pipeline",
+            "description": "The pipeline the file was generated from.",
+            "comment": "For pipelines, see pipeline.json for available identifiers.",
+            "type": "string",
+            "linkTo": "pipeline"
+        },
+        "steps": {
+            "title": "Analysis steps" ,
+            "description": "The runs of the step the file was generated from.",
+            "type": "array",
+            "items": {
+                "title": "Analysis step",
+                "description": "The run instance of the step the file was generated from.",
+                "comment": "For steps, see analysis_step_run.json (and analysis_step.json) - may be virtual",
+                "type": "string",
+                "linkTo": "analysis_step_run"
+            }
```

Comment:
@lrowe workflow_run will only be instatiated for pipelines that we run. Should figure out a better way for lab pipelines then otherwise we could have fantom workflow_run's as well. Will talk to Ben tommorow.